sys/sys/efi.h

```diff
@@ -54,11 +54,26 @@ enum efi_reset {
 typedef uint16_t	efi_char;
 typedef unsigned long efi_status;
 
+/*
+ * Currently no pure-capability EFI exists so we have to shim to a legacy
+ * integer address based interface. Rather than mess with the structs we can
+ * make use of the pointer_interpretation pragma to turn the pointers into
```

Comment:
Finally, a use case for the #pragma. :)

Reply:
Yeah, though I'm not sure this one motivates its existence...

The old compartmentalised tcpdump code had the pragma too, but in that case for cheaply making all the interfaces take capabilities for hybrid.